README.md

```diff
@@ -1,83 +1,250 @@
+Simple emoji support for Node.js.
 
-_simple emoji support for node.js projects_
+[![NPM Badge](https://nodei.co/npm/node-emoji.png)](https://npmjs.com/package/node-emoji)
 
-![node-emoji example](https://i.imgur.com/yIo5Uux.png)
+## Install
 
-**Help wanted:** We are looking for volunteers to maintain this project, if you are interested, feel free to contact me at [me@omnidan.net](mailto:me@omnidan.net)
+```sh
+npm install node-emoji
+```
 
-## Installation
-To install `node-emoji`, you need [node.js](http://nodejs.org/) and [npm](https://github.com/npm/npm#super-easy-install). :rocket:
+## Usage
 
-Once you have that set-up, just run `npm install --save node-emoji` in your project directory. :ship:
+```js
+const emoji = require('node-emoji')
 
-You're now ready to use emoji in your node projects! Awesome! :metal:
+emoji.get('unicorn')
+//=> 🦄
 
-## Usage
-```javascript
-var emoji = require('node-emoji')
-emoji.get('coffee') // returns the emoji code for coffee (displays emoji on terminals that support it)
-emoji.which(emoji.get('coffee')) // returns the string "coffee"
-emoji.get(':fast_forward:') // `.get` also supports github flavored markdown emoji (http://www.emoji-cheat-sheet.com/)
-emoji.emojify('I :heart: :coffee:!') // replaces all :emoji: with the actual emoji, in this case: returns "I ❤️ ☕️!"
-emoji.random() // returns a random emoji + key, e.g. `{ emoji: '❤️', key: 'heart' }`
-emoji.search('cof') // returns an array of objects with matching emoji's. `[{ emoji: '☕️', key: 'coffee' }, { emoji: ⚰', key: 'coffin'}]`
-emoji.unemojify('I ❤️ 🍕') // replaces the actual emoji with :emoji:, in this case: returns "I :heart: :pizza:"
-emoji.find('🍕') // Find the `pizza` emoji, and returns `({ emoji: '🍕', key: 'pizza' })`;
-emoji.find('pizza') // Find the `pizza` emoji, and returns `({ emoji: '🍕', key: 'pizza' })`;
-emoji.hasEmoji('🍕') // Validate if this library knows an emoji like `🍕`
-emoji.hasEmoji('pizza') // Validate if this library knowns a emoji with the name `pizza`
-emoji.strip('⚠️ 〰️ 〰️ low disk space') // Strips the string from emoji's, in this case returns: "low disk space".
-emoji.replace('⚠️ 〰️ 〰️ low disk space', (emoji) => `${emoji.key}:`) // Replace emoji's by callback method: "warning: low disk space"
+emoji.get(':unicorn:')
+//=> 🦄
+
+emoji.which('🦄')
+//=> "unicorn"
+
+emoji.emojify('I :heart: :coffee:!')
+//=> "I ❤️ ☕️!"
+
+emoji.random()
+//=> { name: 'house', emoji: '🏠' }
+```
+
+## API
+
+### emoji.get(name)
+
+Get an emoji from an emoji name.
+
+#### name
+
+Type: `string`
+
+The name of the emoji to get.
+
+```js
+const emoji = require('node-emoji')
+
+console.log(emoji.get('unicorn'))
+//=> '🦄'
+```
+
+### emoji.which(emoji)
+
+Get an emoji name from an emoji.
+
+#### emoji
```

Comment:
It's an accessibility oddity to have headings with the same name (and also linking then only works for one). Should look at getting each of these unique text values...